janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -122,6 +122,11 @@
             "This string should be formatted as a natural number followed by the lowercase letter " +
             "\"s\", e.g. 3s or 60s.", ConfigOption.Type.MASKABLE, "30s");
 
+    public static final ConfigOption<Integer> MAX_RETRIES_IN_MILLIS =
+            new ConfigOption<>(ELASTICSEARCH_NS, "max-retries-in-millis",
```

Comment:
How about `MAX_RETRY_TIMEOUT/max-retry-timeout`? It looks like other configuration items here and in GraphDatabaseConfiguration include the units in the description but not the name/key.